docker-compose.override.yml

```diff
@@ -14,7 +14,9 @@ services:
       - DJANGO_OIDC_USERNAME_CLAIM=preferred_username
     volumes:
       - ./:/app
-    command: /bin/sh -c "wait-for-it.sh -t 60 db:5432 -- ./manage.py migrate && ./manage.py runserver 0.0.0.0:80"
+    command: /bin/sh -c "wait-for-it.sh -t 60 db:5432 -- ./manage.py migrate && ./manage.py serve --port 80 --req-queue-len=${HURRICANE_REQ_QUEUE_LEN:-250} --autoreload --static"
```

Comment:
Why not just run the `cmd.sh` as it's basically doing the same apart from the `--autoreload --static` flags, which could easily be added if in "dev" mode